apps/form-builder/src/components/FormSubmissions/TextQuestionAnalytics.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  FormQuestionResponse,
+  FormSubmissionResponse,
+} from '@hack4impact-utk/internal-models';
+import * as React from 'react';
+import List from '@mui/joy/List';
+import ListItem from '@mui/joy/ListItem';
+import ListItemContent from '@mui/joy/ListItemContent';
+import ListItemButton from '@mui/joy/ListItemButton';
+import Sheet from '@mui/joy/sheet';
+
+//Takes in the question and response as props
+interface Props {
+  question: FormQuestionResponse;
+  responses: FormSubmissionResponse[];
+}
+
+//function that displays the list
+export default function TextQuestionAnalytics({ question, responses }: Props) {
+  //gets the specific submission to a question from each response from a single form
+  function getSubmission() {
```

Comment:
This function is getting multiple answers, so I suggest renaming to `getAnswers()`.